Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.5
```

Comment:
Sure! Would you be up for accept another, separate pull request that adds them to a `docker` directory along with a section in the readme that documents how how to run some TBD tests for this library?